force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +69,35 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
-        
+        List <Boolean> stringCheck = new List <Boolean>();
+        Boolean sales = false;
+        Boolean force = false;
+        Boolean awesome = false;
         // Check if the string starts with 'Sales'
-        
+        if(input.startsWith('Sales')){
+            sales = true;
+            stringCheck.add(sales);
+        }
+        else {
+            stringCheck.add(sales);
+        }
         // Check if the string ends with 'Force'
-        
+        if(input.endsWith('Force')){
+            force = true;
+            stringCheck.add(force);
+        }
+        else {
+            stringCheck.add(force);
+        }
         // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        if(input.contains('Awesome')){
+            awesome = true;
+            stringCheck.add(awesome);
+        }
+        else {
+            stringCheck.add(awesome);
+        }
+        return stringCheck; // Replace null with the variable you used to store the result
```

Comment:
i would challenge you to do this over with a lot less code. When we value something  with an expression that returns a boolean there is no need to declare all the variables first as False and use  if statements. 

If Name = = Frank is only going to return true or false. So you can do something like this. 

 booleans.add(input.contains('Awesome'));